### PR TITLE
Add skeleton for 2021 GSOC students post

### LINKED ISSUE #176: Add GSOC 2021 student post.
Make a post detailing students and student projects for 2021 GSOC!

### PR DESCRIPTION
**Pull request by hmacdope**
Hi all, This is the skeleton for the 2021 GSCO student post. Fixes #176.

The students (@ojeda-e and @orioncohen) now need to fill out their info, along with a picture (if you like). Students should also probably sort out their blog and add it to their little bio.

To do this you will need to PR against this branch. Hopefully you have permissions to do this, otherwise let us know in this PR or in #176. 

For reference [this](https://www.mdanalysis.org/2020/05/20/gsoc-students/) was the post from 2020.

@MDAnalysis/gsoc-mentors please let me know if I have missed anything.

### CODE REVIEW

**Comment by hmacdope** [comment]
Note we need to squash history here.

**Comment by orbeckst** [comment]
nice work @hmacdope

**Comment by hmacdope** [comment]
Ready for review @MDAnalysis/gsoc-mentors.

**Review by orbeckst — COMMENTED** [review]
Looks good to me; would be nice to enable images of the students. 

Perhaps also link to their academic homes (lab pages)?

**Comment by hmacdope** [comment]
Done @orbeckst, just waiting for @orioncohen blog link :)

**Comment by orioncohen** [comment]
> Done @orbeckst, just waiting for @orioncohen blog link :)

I'll get a blog up by the end of the week!

**Comment by orbeckst** [comment]
Looking good, I hope to see it up soon & on twitter.

**Comment by hmacdope** [comment]
@MDAnalysis/gsoc-mentors all links included now

File: _posts/2021-05-19-gsoc-students.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Estefania is a Ph.D. candidate in Biophysical Chemistry at The University of Calgary, Canada in the research group of [Peter Tieleman][tieleman-group]. Her research work is focused on membrane curvature induced by ABC transporters, a superfamily of transmembrane proteins involved in cancer and antibiotic resistance. A typical day for Estefania includes running Coarse-Grained (CG) MD simulations using the [Martini force field][martini-url], reading literature on ABCs, and working on cool data visualization workflows. In her free time, she enjoys camping and road tripping in the Canadian Rockie Mountains and going for long bike rides. 
+
+Estefania can be found on github as [@ojeda-e][ojeda-git] and on twitter as [@ebojeda][ojeda-twitter].
```
Comment (orbeckst) [inline _posts/2021-05-19-gsoc-students.md:22]: The GH markdown parser automatically links `@-handles`
```suggestion
Estefania can be found on github as @ojeda-e] and on twitter as [@ebojeda][ojeda-twitter].
```

File: _posts/2021-05-19-gsoc-students.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Orion is a Ph.D. student at the University of California Berkeley, working
+with [Dr. Kristin Persson][persson-group] at the Lawrence Berkeley National Laboratory. His
+work leverages high-throughput chemical simulations and machine learning to
+discover new materials for Lithium-ion batteries. Orion is passionate about
+making science more accessible, reproducible, and efficient with powerful
+open-source software. If he isn't toiling at his computer, Orion is probably
+playing board games, camping, or relaxing in the temperate Berkeley sun.
+
+Orion is on Github as [@orioncohen][orion-git] and on twitter as [@orion__archer][orion-twitter].
```
Comment (orbeckst) [inline _posts/2021-05-19-gsoc-students.md:54]:
```suggestion
Orion is on Github as @orioncohen and on twitter as [@orion__archer][orion-twitter].
```

File: _posts/2021-05-19-gsoc-students.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+He will be sharing his experience with GSoC on [his website][orion-blog].
+
+— @richardjgowers @IAlibay @fiona-naughton @orbeckst @lilyminium @hmacdope (mentors)
+
+[gsoc]: https://summerofcode.withgoogle.com
+[mda-gsoc]: https://summerofcode.withgoogle.com/organizations/6414449348444160/
+[martini-url]: http://cgmartini.nl/
+[ojeda-git]: https://github.com/ojeda-e
```
Comment (orbeckst) [inline _posts/2021-05-19-gsoc-students.md:63]: Not needed with my other suggestions.
```suggestion
```

File: _posts/2021-05-19-gsoc-students.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+[ojeda-blog]: https://ojeda-e.github.io/
+[ojeda-twitter]: https://twitter.com/ebojeda
+[ojeda-project]: https://summerofcode.withgoogle.com/projects/#5098282306502
+[tieleman-group]: https://ucalgary.ca/biocomputing/home
+[orion-git]: https://github.com/orioncohen
```
Comment (orbeckst) [inline _posts/2021-05-19-gsoc-students.md:68]:
```suggestion
```

**Review by orbeckst — APPROVED** [review]
Looking good.

Optional suggestions on reducing number of links by using the automatic GitHub handle expansion in the jekyll markdown interpreter on GitHub.

**Review by IAlibay — APPROVED** [review]

**Review by lilyminium — APPROVED** [review]
LGTM. Thanks @orioncohen and @ojeda-e, and thanks @hmacdope for putting it together.

**Comment by lilyminium** [comment]
Do you want to do the tweet honours @IAlibay ?

**Comment by IAlibay** [comment]
Sorry @orbeckst I just saw two approvals and just merged (didn't se eyour comments come through).

I'll do a cleanup PR to add in your changes.

**Comment by IAlibay** [comment]
> Do you want to do the tweet honours @IAlibay ?

Yeah will do.

**Pull request merged**